sdl_android/src/main/java/com/smartdevicelink/protocol/enums/BsonTags.java

```diff
@@ -0,0 +1,12 @@
+package com.smartdevicelink.protocol.enums;
+
+public class BsonTags {
```

Comment:
We should make this a little better organized; creating subclasses in this class would be fine. Open to suggestions. Also adding descriptions to the constants.